src/lib/formats/json/parser.ts

```diff
@@ -8,10 +8,11 @@ const ValidationError = require("../../bindings/http/validation/validation_error
 const invalidPayloadTypeError = new ValidationError("invalid payload type, allowed are: string or object");
 const nullOrUndefinedPayload = new ValidationError("null or undefined payload");
 
-const asJSON = (v) => (isString(v) ? JSON.parse(v) : v);
+const asJSON = (v: any) => (isString(v) ? JSON.parse(v) : v);
```

Comment:
I believe this could be 
```suggestion
const asJSON = (v: string|object) => (isString(v) ? JSON.parse(v) : v);
```
Since these are the only two valid options.